apps/web/src/pages/get-started/components/timeline/Timeline.styles.ts

```diff
@@ -0,0 +1,49 @@
+import { createStyles, MantineTheme } from '@mantine/core';
+import { ExpandStepsTypeEnum } from './Timeline';
+
+interface ITimelineParams {
+  expandSteps: ExpandStepsTypeEnum;
+}
+
+export default createStyles((theme: MantineTheme, { expandSteps }: ITimelineParams, getRef) => {
+  const itemContent = getRef('itemContent');
```

Comment:
I am not sure what would be the other way to do it without coupling to mantine, any suggestions? 🙃

Reply:
I haven't used something quite like `createStyles` before, but it using an object instead of just pure CSS is probably the challenge here. In this case, I would just propose that we not necessarily use `createStyles` and instead use an emotion `.styled` to determine behavior. It's okay if there's a bit of repetition / copy-paste IMO

Reply:
I guess another way you can target elements by their classes ;)

Reply:
Maybe I am missing something but targeting mantine elements without mantine selectors is a thing that is super hard to do.

Reply:
@djabarovgeorge I believe you can use [these static selectors](https://v5.mantine.dev/core/timeline/?t=styles-api), but either way is still coupled with Mantine, so I'm fine with leaving this for now.